spec/factories/organizations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :organization do
+    user
```

Comment:
Please, add `association :user, factory: :user` insted of `user `.

Reply:
Why?

Reply:
It does the same.
🤔 I don't know this feature.